Library/Formula/awf.rb

```diff
@@ -0,0 +1,30 @@
+class Awf < Formula
+  desc "awf (A Widget Factory) is a theme preview application for gtk2 and gtk3"
+  homepage "https://github.com/valr/awf"
+  url "https://github.com/valr/awf/archive/v1.3.0.tar.gz"
+  sha256 "911ded8a307beecbb8ee1dab489fa7b1aa9d7965cb34bd938482220505814fa4"
+  head "https://github.com/valr/awf.git"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "gtk+"
+  depends_on "gtk+3"
+
+  def install
+    inreplace "src/awf.c", "/usr/share", "#{HOMEBREW_PREFIX}/share"
```

Comment:
Doesn't that end up in the Cellar, where it is dark and damp and no other live ?